src/provider/ContextProvider.js

```diff
@@ -0,0 +1,18 @@
+import { combineComponents } from './combineComponents';
+
+import { I18nextProvider } from 'react-i18next';
+import i18n from '../i18n';
+import { ThemeProvider } from '../components/publiq-ui/ThemeProvider';
+import { CookiesProvider } from 'react-cookie';
+import { queryCache, ReactQueryCacheProvider } from 'react-query';
+
+const providers = [
+  ({ children }) => I18nextProvider({ children, i18n }),
+  ThemeProvider,
+  CookiesProvider,
+  ({ children }) => ReactQueryCacheProvider({ children, queryCache }),
+];
+
+const ContextProvider = combineComponents(...providers);
```

Comment:
I think I would prefer to make it a little bit more declarative like this.

https://medium.com/@juliuskoronci/avoid-a-long-list-of-react-providers-c45a269d80c1